### PR TITLE
docs: Hub Router API

### LINKED ISSUE #20: Hub-Router docs

### PR DESCRIPTION
**Pull request by rolsonquadras**
closes #20 

Signed-off-by: Rolson Quadras <rolson.quadras@securekey.com>

### CODE REVIEW

**Review by sudeshrshetty — APPROVED** [review]

File: docs/api.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+      "label":"hub-router",
+      "service":[
+         {
+            "ID":"58fe0316-ea25-4db4-b4b2-946a9b75cb3e",
+            "Type":"did-communication",
+            "Priority":0,
+            "RecipientKeys":[
+               "8XQawExAm8s2N1U9i4zBWEUmeqBDW3rfDLnqoSn92acc"
+            ],
+            "RoutingKeys":null,
```
Comment (sandrask) [inline docs/api.md]: Should you omit routing keys if empty - not sure about spec :)

**Review by sandrask — APPROVED** [review]

File: docs/api.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+            "ServiceEndpoint":"wss://hub-router.example.com:10202",
+            "Properties":null
```
Comment (troyronda) [inline docs/api.md]: also here.

File: docs/api.md (hunk 1 above)
Comment (rolsonquadras) [reply docs/api.md]: [Removed](https://github.com/trustbloc/hub-router/compare/1b2fa7cdbdf6105594a6bd9f7f8b15431cdfcb32..2e9d8f798132690b012126918748d9e0ae6b6570).

File: docs/api.md (hunk 2 above)
Comment (rolsonquadras) [reply docs/api.md]: [Removed](https://github.com/trustbloc/hub-router/compare/1b2fa7cdbdf6105594a6bd9f7f8b15431cdfcb32..2e9d8f798132690b012126918748d9e0ae6b6570)

File: docs/api.md (hunk 1 above)
Comment (rolsonquadras) [reply docs/api.md]: Thanks, I missed to validate the sample response :)

**Pull request merged**